resources/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:18.04 as dev
+
+ARG RUST_TOOLCHAIN="1.40.0"
+ARG CLH_SRC_DIR="/cloud-hypervisor"
+ARG CLH_BUILD_DIR="$CLH_SRC_DIR/build"
+ARG CARGO_REGISTRY_DIR="$CLH_BUILD_DIR/cargo_registry"
+ARG CARGO_GIT_REGISTRY_DIR="$CLH_BUILD_DIR/cargo_git_registry"
+
+ENV CARGO_HOME=/usr/local/rust
+ENV RUSTUP_HOME=/usr/local/rust
```

Comment:
nit: Maybe `RUSTUP_HOME=$CARGO_HOME` would be more appropriate.

Reply:
Thanks, will fix.